src/index.html

```diff
@@ -1,13 +1,51 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="container">
+      <div class="card container__card" data-qa="card">
+        <a href="#!" class="card__link">
+          <img
+            src="images/imac.jpeg"
+            alt="Moyo-Product-imac/APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+            class="card__img"
+          >
+        </a>
+        <a href="#!" class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </a>
+        <div class="card__product-code">Product code: 195434</div>
+        <a href="#!" class="card__stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <span class="card__stars-review">Reviews: 5</span>
+        </a>
+
+        <div class="card__prices">
+          <div class="card__price">Price:</div>
+          <div class="card__value">$2,199</div>
+        </div>
+        <a
+          href="#!"
+          class="card__button"
+          data-qa="hover"
+        >
+          Buy
+        </a>
+      </div>
+    </div>
+  </body>
 </html>
```

Comment:
Why you do all elements like links? It's only needed for button

Reply:
This is not a necessary decision here but I tried to interpret real product-card where user always can click on image or title in card and get to another page.
We can see it on all websites such as  Amazon, Rozetka etc

Reply:
I find it more useful and comfortable for supposed customer and I tried to make it in this task :) As well, in Checklist i can't find any info about prohibition of using links in title or image, I don't think that this case damaged our semantic because tag `a` is't the same thing as `div` tag in this situation